packages/components/ng-ovh-feature-flipping/src/provider.js

```diff
@@ -0,0 +1,21 @@
+import FeatureFlipping from './service';
+
+export default class NgOvhFeatureFlippingProvider {
+  constructor() {
+    this.applicationName = null;
+  }
+
+  /**
+   * Set the application name that will request the 2API feature availability.
+   *
+   * @param {string} applicationName The current application name
+   */
+  setApplicationName(applicationName) {
```

Comment:
Couldn't be turned into a setter?

```suggestion
  set applicationName(applicationName) {
```